### ISSUE TITLE
Update docker image

### ISSUE THREAD

**Issue by alfonder** [BoKKeR/RSS-to-Telegram-Bot#26]
DockerHub image needs to be updated according to the latest changes.
Thanks.

**Comment by BoKKeR** [comment]
Sorry, with docker removing auto builds on the free tier I will have to come up with a GitHub action before I can build the new version

**Comment by alfonder** [comment]
Yes, I know it. I needed to integrate GitHub Actions in my project too.

**Comment by BoKKeR** [comment]
Should be fixed now, I pushed latest/dev. Thanks for all the work :)
